README.md

```diff
@@ -10,26 +10,39 @@ The documentation is built using Markdown pages, which can be converted into a n
 
 ## Build dependencies
 
-The documentation site uses [Hugo](https://gohugo.io/), and specifically uses features found in the extended version of Hugo, v0.76.0 and later. See `.tool-versions` for the currently used version
+The documentation site uses [Hugo](https://gohugo.io/) including the extended version of Hugo. Version v0.122 is used per (see #https://github.com/medic/cht-docs/issues/1302) as denoted in `.tool-versions` [file](https://github.com/medic/cht-docs/blob/main/.tool-versions).
 
-[asdf](https://asdf-vm.com/guide/getting-started.html) is the recommended way to manage `hugo` and `golang` versions for local development.
+To proceed, please choose one of the three options below of Docker, native packages or using `asdf` then proceed to ["Run Hugo" below](#run-hugo) to start your local instance.
 
-After installing it run:
+### Docker
+
+Running `hugo` locally using Docker is likely the easiest way to update the docs.  You only need to ensure you have [Docker and Docker compose installed](https://docs.docker.com/compose/install/).
+
+### Native packages
+
+If you'd like to install packages directly on your workstation, follow the [installation instructions for your Operating System](https://gohugo.io/getting-started/installing/), and be sure to get the extended version. Most users will be able to simply install using their native package manager like `brew`, `apt` or `snap`.
+
+### `asdf` version manager
+
+[asdf](https://asdf-vm.com/guide/getting-started.html) is another way to manage `hugo` and `golang` versions for local development.
+
+After installing `asdf`, run:
 ```shell
 asdf plugin add golang
 asdf plugin add hugo
 ```
-### Installing Hugo Manually
 
-If using `asdf` fails you can try installing Hugo manually. To install, follow the [installation instructions for your Operating System](https://gohugo.io/getting-started/installing/), and be sure to get the extended version. Most users will be able to simply install using their native package manager like `brew`, `apt` or `snap`.
+### Run Hugo
 
-## Building the Documentation
+For both native packages or `asdf`, you can start your local CHT Docs instance by:
 
 1. Clone the `cht-docs` repo
-1. From the command line, navigate to the local directory where you cloned the repo
-1. Type `hugo server`
-1. Preview your site in your browser at: http://localhost:1313/
-1. As you make changes to the site, your browser will automatically reload your local dev site so you can easily and quickly see your changes.
+2. From the command line, navigate to the local directory where you cloned the repo
+3. To start `hugo` run:
+   * `hugo server` for `asdf` and native packages **OR**
+   * `docker compose up` for Docker compose
```

Comment:
So, one downside to running via Docker Compose is that it will end up leaving a stopped container sitting on your host _forever_ (unless you manually remove it).  It is not a huge deal, but it seems like, out of politeness, we should provide instructions for how to delete the container.

Alternatively, we could suggest running the container via the `docker compose run --rm -it -p 1313:1313 hugo` command and then I think the container will be automatically removed when the command is stopped.

Reply:
yeah - love the ephemeral approach!  That said, testing it, I get ~20 second start up ever time for the `--rm` approach and only that on the first start for the vanila compose.  It's otherwise ~3 seconds on subsequent starts.  

I propose we keep it as I have, add an explicit name in the compose file and call out how to delete it.  Done in 21e999fc